shard.yml

```
@@ -9,8 +9,8 @@ authors:
 development_dependencies:
   ameba:
     github: crystal-ameba/ameba
-    version: ~> 0.13.0
+    version: ~> 0.14.2
 
-crystal: ">= 0.33.0"
+crystal: 1.0.0
```

Pretty sure this means that as soon as `1.0.1` was released it would no longer work.

This is how I've typically seen the version set:
```suggestion
crystal: ">= 0.33.0, < 2.0.0"
```

---

I always thought that Crystal that it worked like a "minimal required version".

---

@matthewmcgarvey Defining the upper boundary shouldn't be required.